java/test/jmri/jmrit/display/switchboardEditor/SwitchboardEditorTest.java

```diff
@@ -79,6 +81,16 @@ public void testSetDefaultTextColor() {
         Assert.assertEquals("Default Text Color after Set", ColorUtil.ColorPink, swe.getDefaultTextColor());
     }
 
+    @Test
+    public void checkHelpMenuExists() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        e.setVisible(true);
+        EditorFrameOperator jfo = new EditorFrameOperator(e);
+        JMenuOperator jmo = new JMenuOperator(jfo, Bundle.getMessage("MenuHelp"));
+        Assert.assertNotNull("Help Menu Exists", jmo);
+        Assert.assertEquals("Menu Item Count", 8, jmo.getItemCount());
```

Comment:
See my previous comment.  With the last appveyor run, this test failed for SwitchboardEditor but not for the rest.  SwitchboardEditor is reporting the same number of entries as the rest, so this change needs to be reverted.

Reply:
I didn't change anything in any help menu… (esp. not in SwitchboardEditor); I don't know why that was failing. I can undo the fix… but then the checks will fail again. What to do?

Reply:
I'm seeing this in the console:
Exception in thread "AWT-EventQueue-0" java.lang.ClassCastException
	at javax.swing.LayoutComparator.compare(LayoutComparator.java:75)
	at javax.swing.LayoutComparator.compare(LayoutComparator.java:42)
	at java.util.TimSort.countRunAndMakeAscending(TimSort.java:360)
	at java.util.TimSort.sort(TimSort.java:234)
	at java.util.Arrays.sort(Arrays.java:1512)
	at java.util.ArrayList.sort(ArrayList.java:1454)
	at java.util.Collections.sort(Collections.java:175)
	at javax.swing.SortingFocusTraversalPolicy.enumerateAndSortCycle(SortingFocusTraversalPolicy.java:172)
	at javax.swing.SortingFocusTraversalPolicy.getFocusTraversalCycle(SortingFocusTraversalPolicy.java:143)
	at javax.swing.SortingFocusTraversalPolicy.getFirstComponent(SortingFocusTraversalPolicy.java:500)
	at javax.swing.LayoutFocusTraversalPolicy.getFirstComponent(LayoutFocusTraversalPolicy.java:166)
	at javax.swing.SortingFocusTraversalPolicy.getDefaultComponent(SortingFocusTraversalPolicy.java:593)
	at java.awt.FocusTraversalPolicy.getInitialComponent(FocusTraversalPolicy.java:169)
	at java.awt.DefaultKeyboardFocusManager.dispatchEvent(DefaultKeyboardFocusManager.java:420)
	at java.awt.Component.dispatchEventImpl(Component.java:4760)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

Reply:
Something changed the item count in the help menu from 8 to 10, and it changed it in all 4 of the editors, that's what the appveyor tests show.  If you didn't change it, Then something else in this PR did, and that's a problem.

I see no indication in the stacktrace you provided that says there is s problem in any of the checkHelpMenuExists tests.

Reply:
> Then something else in this PR did
You can see the files this PR touches: <https://github.com/JMRI/JMRI/pull/4353/files>
Nothing there for the switchboard (except the Test I modified to pass).

Reply:
Ok, I just pulled down the JMRI/master and ran the SwitchboardEditorTest… and it FAILS:

Testcase: checkHelpMenuExists(jmri.jmrit.display.switchboardEditor.SwitchboardEditorTest):	FAILED
Menu Item Count expected:<10> but was:<8>
junit.framework.AssertionFailedError: Menu Item Count expected:<10> but was:<8>
	at jmri.jmrit.display.AbstractEditorTestBase.checkHelpMenuExists(AbstractEditorTestBase.java:57)

So it's NOT something in my PR.

Reply:
after https://github.com/JMRI/JMRI/pull/4353/commits/adf6b7713a830fde96ac37b614591c26f0877cbb, the appveyor build got past these tests and failed on:

mvn : [jacoco:report] Rule violated for package jmri.jmrix.anyma.configurexml: classes covered count is 0, but expected minimum is 1
At line:1 char:1
+ mvn test -P appveyor --batch-mode
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: ([jacoco:report]...ed minimum is 1:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
[INFO] ------------------------------------------------------------------------

You put tests in that package, but they aren't being called.
There also are no tests for the jmri.jmrix.anyma package, which would be where you would call the package test for the group of tests above.